Dockerfile

```diff
@@ -1,61 +1,72 @@
-FROM fedora
+FROM alpine:3.5
 
-MAINTAINER Guillaume Scheibel <guillaume.scheibel@gmail.com>
+LABEL MAINTAINER="Guillaume Scheibel <guillaume.scheibel@gmail.com>"
+LABEL MAINTAINER="Damien DUPORTAL <damien.duportal@gmail.com>"
 
-ENV JAVA_HOME /jdk1.8.0_112
-ENV PATH $PATH:$JAVA_HOME/bin:/fopub/bin
-ENV BACKENDS /asciidoctor-backends
-ENV GVM_AUTO_ANSWER true
-ENV ASCIIDOCTOR_VERSION "1.5.5"
+ENV JAVA_HOME=/usr/lib/jvm/default-jvm \
+  PATH=${PATH}:${JAVA_HOME}/bin:/fopub/bin \
+  BACKENDS=/asciidoctor-backends \
+  GVM_AUTO_ANSWER=true \
+  ASCIIDOCTOR_VERSION="1.5.5"
 
-RUN dnf install -y tar \
-    make \
-    gcc \
-    ruby \
-    ruby-devel \
-    rubygems \
+RUN apk --update --no-cache add \
+    asciidoctor \
+    bash \
+    build-base \
+    curl \
     graphviz \
-    rubygem-nokogiri \
+    jpeg \
+    jpeg-dev \
+    openjdk8 \
+    patch \
+    python \
+    python-dev \
+    py-pillow \
+    py-setuptools \
+    ruby \
+    ruby-dev \
+    tar \
+    ttf-liberation \
     unzip \
     findutils \
     which \
     wget \
-    python-devel \
-    zlib-devel \
-    libjpeg-devel \
-    redhat-rpm-config \
-    patch \
-    sudo \
-  && dnf clean packages \
-  && (curl -s -k -L -C - -b "oraclelicense=accept-securebackup-cookie" http://download.oracle.com/otn-pub/java/jdk/8u112-b15/jdk-8u112-linux-x64.tar.gz | tar xfz -) \
+    zip \
+    zlib-dev \
   && mkdir /fopub \
-  && curl -L https://api.github.com/repos/asciidoctor/asciidoctor-fopub/tarball | tar xzf - -C /fopub/ --strip-components=1 \
-  && touch /tmp/empty.xml \
-  && fopub /tmp/empty.xml \
-  && rm /tmp/empty.xml \
-  && gem install --no-ri --no-rdoc asciidoctor --version $ASCIIDOCTOR_VERSION \
+  && curl -L -s https://api.github.com/repos/asciidoctor/asciidoctor-fopub/tarball | tar xzf - -C /fopub/ --strip-components=1 \
   && gem install --no-ri --no-rdoc asciidoctor-diagram \
   && gem install --no-ri --no-rdoc asciidoctor-epub3 --version 1.5.0.alpha.6 \
+  && gem install --no-ri --no-rdoc asciidoctor-revealjs \
   && gem install --no-ri --no-rdoc rake \
   && gem install --no-ri --no-rdoc epubcheck --version 3.0.1 \
   && gem install --no-ri --no-rdoc kindlegen --version 3.0.1 \
```

Comment:
We should move to kindlegen 3.0.3. I've verified it (on another project) and it works.